templates/taxi/driver_list.html

```diff
@@ -6,8 +6,13 @@ <h1>Driver list</h1>
     <ul>
       {% for driver in driver_list%}
         <li>
-          <a href="{% url "taxi:driver-detail" pk=driver.id %}">{{ driver.username }}</a>
-          ({{ driver.first_name }} {{ driver.last_name }})
+          {% if user.id == driver.id %}
+            <a href="{% url "taxi:driver-detail" pk=driver.id %}">{{ driver.username }}</a>
+            ({{ driver.first_name }} {{ driver.last_name }}) (Me)
```

Comment:
The logic to identify the current user and append '(Me)' is correctly implemented. Ensure that `user` is available in the template context, typically provided by Django's authentication context processor.